documentation/privatebuildtesting.md

```diff
@@ -1,36 +1,51 @@
 Private runtime build testing
 =============================
 
-Here are some instructions on how to run the diagnostics repo's tests against a locally build private .NET Core runtime based on CoreCLR. These directions will work on Windows, Linux and MacOS. 
+Here are some instructions on how to run the diagnostics repo's tests against a locally built private .NET Core runtime based on CoreCLR. These directions will work on Windows, Linux and MacOS. 
 
-1. Build the runtime repo (see [Workflow Guide](https://github.com/dotnet/runtime/blob/main/docs/workflow/README.md)).
-2. Build the diagnostics repo (see [Building the Repository](../README.md)).
-3. Run the diagnostics repo tests with the -privatebuildpath option.
+1. Build the runtime repo (see [Workflow Guide](https://github.com/dotnet/runtime/blob/main/docs/workflow/README.md)) with `-configuration release -subset clr+libs`. A release build is highly recommended.
```

Comment:
They can still run with a checked or debug build but because some debuggee's are built with the debug runtime and the tests themselves will take a very long time.